iron-dropdown-scroll-manager.d.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   iron-dropdown-scroll-manager.html
+ */
+
+/// <reference path="../iron-overlay-behavior/iron-scroll-manager.d.ts" />
```

Comment:
Should we define the `IronDropdownScrollManager` interface here?

Reply:
Done. Had to do it manually, see `extra-types.d.ts`.